functions/New-DbaAgentJob.ps1

```diff
@@ -0,0 +1,476 @@
+
+.PARAMETER Description
+The description of the job.
+
+.PARAMETER StartStepId
+The identification number of the first step to execute for the job.
+
+.PARAMETER CategoryName
+The category of the job.
+
+.PARAMETER CategoryId
+A language-independent mechanism for specifying a job category.
+
+.PARAMETER OwnerLoginName
+The name of the login that owns the job.
+
+.PARAMETER EventLogLevel
+Specifies when to place an entry in the Microsoft Windows application log for this job.
+Allowed values 0, "Never", 1, "OnSuccess", 2, "OnFailure", 3, "Always"
+The text value van either be lowercase, uppercase or something in between as long as the text is correct.
+
+.PARAMETER EmailLevel
+Specifies when to send an e-mail upon the completion of this job.
+Allowed values 0, "Never", 1, "OnSuccess", 2, "OnFailure", 3, "Always"
+The text value van either be lowercase, uppercase or something in between as long as the text is correct.
+
+.PARAMETER NetsendLevel
+Specifies when to send a network message upon the completion of this job.
+Allowed values 0, "Never", 1, "OnSuccess", 2, "OnFailure", 3, "Always"
+The text value van either be lowercase, uppercase or something in between as long as the text is correct.
+
+.PARAMETER PageLevel
+Specifies when to send a page upon the completion of this job.
+Allowed values 0, "Never", 1, "OnSuccess", 2, "OnFailure", 3, "Always"
+The text value van either be lowercase, uppercase or something in between as long as the text is correct.
+
+.PARAMETER EmailOperatorName
+The e-mail name of the operator to whom the e-mail is sent when EmailLevel is reached.
+
+.PARAMETER NetsendOperator
+The name of the operator to whom the network message is sent.
+
+.PARAMETER PageOperator
+The name of the operator to whom a page is sent.
+
+.PARAMETER DeleteLevel
+Specifies when to delete the job.
+Allowed values 0, "Never", 1, "OnSuccess", 2, "OnFailure", 3, "Always"
+The text value van either be lowercase, uppercase or something in between as long as the text is correct.
+
+.NOTES 
+Original Author: Sander Stad (@sqlstad, sqlstad.nl)
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/New-DbaAgentJob
+
+.EXAMPLE   
+New-DbaAgentJob -SqlServer 'sql1' -JobName 'Job1' -Description 'Just another job'
```

Comment:
Job1 would not need quotes - please either remove quotes or name it 'Example Job' or 'Job 1' so that the quotes are justified 👍